### PR TITLE
Update Onboard-Windows-10-multi-session-device.md

### PR DESCRIPTION
**Pull request by pahuijbr**

### CODE REVIEW

**Review by denisebmsft — APPROVED** [review]
Thank you @pahuijbr

File: windows/security/threat-protection/microsoft-defender-atp/Onboard-Windows-10-multi-session-device.md [hunk 1]
```diff
@@ -13,63 +13,59 @@ ms.topic: article
...
+Familiarize yourself with the [considerations for non-persistent VDI](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-atp/configure-endpoints-vdi#onboard-non-persistent-virtual-desktop-infrastructure-vdi-devices-1). Although [Windows Virtual Desktop](https://docs.microsoft.com/azure/virtual-desktop/overview) does not provide non-persistence options, it does provide ways to use a Windows image that can be used to provision new hosts and redeploy machines. This increases volatility in the environment, and thus impacts what entries are created and maintained in the Microsoft Defender Security Center ([https://securitycenter.windows.com](https://securitycenter.windows.com)), potentially reducing visibility for your security analysts.
```
Comment (adirdidi) [inline windows/security/threat-protection/microsoft-defender-atp/Onboard-Windows-10-multi-session-device.md:30]: This is wrong. The right term is "Microsoft Defender for Endpoint" portal.

File: windows/security/threat-protection/microsoft-defender-atp/Onboard-Windows-10-multi-session-device.md [hunk 2]
```diff
@@ -13,63 +13,59 @@ ms.topic: article
...
 
 > [!NOTE]
-> Depending on your choice of onboarding method, devices can appear in Microsoft Defender for Endpoint portal as either: 
+> Depending on your choice of onboarding method, devices can appear in MMicrosoft Defender Security Center as either:
```
Comment (adirdidi) [inline windows/security/threat-protection/microsoft-defender-atp/Onboard-Windows-10-multi-session-device.md:33]: Typo (double "M") + Wrong term.

File: windows/security/threat-protection/microsoft-defender-atp/Onboard-Windows-10-multi-session-device.md [hunk 3]
```diff
@@ -13,63 +13,59 @@ ms.topic: article
...
 
-Microsoft recommends onboarding Windows Virtual Desktop as a single entry per virtual desktop. This ensures that the investigation experience in the Microsoft Defender Endpoint portal is in the context of one device based on the machine name. Organizations that frequently delete and re-deploy WVD hosts should strongly consider using this method as it prevents multiple objects for the same machine from being created in the Microsoft Defender for Endpoint portal. This can lead to confusion when investigating incidents. For test or non-volatile environments, you may opt to choose differently. 
+Microsoft recommends onboarding Windows Virtual Desktop as a single entry per virtual desktop. This ensures that the investigation experience in the Microsoft Defender Security Center is in the context of one device based on the machine name. Organizations that frequently delete and re-deploy WVD hosts should strongly consider using this method as it prevents multiple objects for the same machine from being created in the Microsoft Defender Security Center. This can lead to confusion when investigating incidents. For test or non-volatile environments, you may opt to choose differently.
```
Comment (adirdidi) [inline windows/security/threat-protection/microsoft-defender-atp/Onboard-Windows-10-multi-session-device.md:37]: Wrong term (twice).

**Comment by adirdidi** [comment]
@denisebmsft Please see my comments. Thanks.

File: windows/security/threat-protection/microsoft-defender-atp/Onboard-Windows-10-multi-session-device.md [hunk 4]
```diff
@@ -13,63 +13,59 @@ ms.topic: article
...
 Use the instructions in [Onboard non-persistent virtual desktop infrastructure VDI devices](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-atp/configure-endpoints-vdi#onboard-non-persistent-virtual-desktop-infrastructure-vdi-devices-1).
 
 Follow the instructions for a single entry for each device.
 
-#### *Scenario 2: Using domain group policy*
+### Scenario 2: Using domain group policy
 This scenario uses a centrally located script and runs it using a domain-based group policy. You can also place the script in the golden image and run it in the same way.
 
-**Download the WindowsDefenderATPOnboardingPackage.zip file from the Windows Defender Security Center**
+#### Download the WindowsDefenderATPOnboardingPackage.zip file from the Windows Defender Security Center
```
Comment (adirdidi) [inline windows/security/threat-protection/microsoft-defender-atp/Onboard-Windows-10-multi-session-device.md:60]: "Windows Defender Security Center" -> "Microsoft Defender for Endpoint portal".

File: windows/security/threat-protection/microsoft-defender-atp/Onboard-Windows-10-multi-session-device.md [hunk 5]
```diff
@@ -13,63 +13,59 @@ ms.topic: article
...
 1. Open the VDI configuration package .zip file (WindowsDefenderATPOnboardingPackage.zip)  
     - In the Microsoft Defender Security Center navigation pane, select **Settings** > **Onboarding**.
```
Comment (adirdidi) [inline windows/security/threat-protection/microsoft-defender-atp/Onboard-Windows-10-multi-session-device.md:62]: Needs to change as well.

**Pull request merged**